bugrap/src/main/java/org/vaadin/bugrap/ui/ReportsView.java

```diff
@@ -97,65 +104,37 @@ private void refreshGridContent() {
 			gridReports.getSelectedItems().addAll(selectedReports);
 	}
 
-	private void revertChanges() {
-		model.revertChanges(gridReports.getSelectedItems());
-	}
-
-	private void initializeBinder() {
-		binder.bind(cmbEditPriority, Report::getPriority, Report::setPriority);
-		binder.bind(cmbEditType, Report::getType, Report::setType);
-		binder.bind(cmbEditStatus, Report::getStatus, Report::setStatus);
-		binder.bind(cmbEditAssignedTo, Report::getAssigned, Report::setAssigned);
-		binder.bind(cmbEditVersion, Report::getVersion, Report::setVersion);
-		binder.bind(txtReportDescription, Report::getDescription, Report::setDescription);
-	}
-
 	private void onReportSelected(SelectionEvent<Report> evt) {
 		int selectedItemCount = evt.getAllSelectedItems().size();
 		if (selectedItemCount == 0) {
 			vsplit.setSplitPosition(SLIDER_NOSELECTION);
 		}else {
-			if (selectedItemCount == 1) {
-				processSingleSelection(evt.getFirstSelectedItem().get());
-				vsplit.setSplitPosition(SLIDER_SINGLESELECTION);
-			} else if (selectedItemCount > 1) {
-				processMultiSelection(evt.getAllSelectedItems());
-				vsplit.setSplitPosition(SLIDER_MULTISELECTION);
-			} 
+			binder.setBean(ReportUtil.getCommonFields(evt.getAllSelectedItems()));
```

Comment:
Even in the mass mode the state should still be retrieved from the Model (and not the binder).

Consider a way of not having to deal with many selected reports instances in the View, but just one, as it was before. There are different ways of doing it.